### ISSUE TITLE
Cannot get config when running a binary from a package

### ISSUE THREAD

**Issue by deinspanjer** [doodadjs/npm-package-config#1]
With a package.json such as:
```
{ "name": "foo", "bin": "./lib/cli.js", "config": { "bar": "one", "baz": 2 } }
```

and a cli.js of:
```
#!/usr/bin/env node

'use strict';

const config = require('npm-package-config').list()
console.log(config)
```

Trying to run `foo` gives you the package info for npm-package-config rather than for foo.

**Comment by doodadjs** [comment]
@deinspanjer I've created a new branch 0.1.11 with the fix. Please confirm so that I can merge and publish.

**Comment by deinspanjer** [comment]
Okay! I was able to pull down the branch and it is letting me see the config and override them.  Thank you!

**Comment by doodadjs** [comment]
@deinspanjer  I've reviewed code a little bit and published it on npm under version 0.2.0.
